spec/lucky/memoize_spec.cr

```diff
@@ -23,11 +29,39 @@ describe "memoizations" do
     object = ObjectWithMemoizedMethods.new
 
     object.method_1.should eq "method_1"
-    2.times { object.method_1 }
+    9.times { object.method_1 }
```

Comment:
Could you keep these and others as 2? I think using the bare minimum to find a failure is good because `9` looks like it is a "magic number"